<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectFilesPage</class>
 <widget class="QWidget" name="SelectFilesPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lbUnknownDesc">
     <property name="text">
      <string>If you check the following field, sentences containing words with unknown terminals will also be imported (words with more than one terminal will still cause the sentence to be ignored).</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbIncludeUnknown">
     <property name="text">
      <string>Also import unknown sentences</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lbEncoding">
       <property name="text">
        <string>Encoding:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KComboBox" name="cbEncoding"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="KUrlRequester" name="urFileToAdd">
     <property name="filter">
      <string>Textfiles *.txt</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="KEditListBox" name="elbFiles">
     <property name="title">
      <string>Files to import</string>
     </property>
     <property name="buttons">
      <set>KEditListBox::Add|KEditListBox::Remove</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KEditListBox</class>
   <extends>QGroupBox</extends>
   <header>keditlistbox.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
